homeassistant/components/todoist/calendar.py

```diff
@@ -279,23 +283,24 @@ async def add_reminder(reminder_due: dict):
     )
 
 
-class TodoistProjectEntity(CalendarEntity):
+class TodoistProjectEntity(CoordinatorEntity, CalendarEntity):
```

Comment:
```suggestion
class TodoistProjectEntity(CoordinatorEntity[TodoistCoordinator], CalendarEntity):
```

Reply:
Thanks one of my previous commits reverted this change.